bndtools.core/src/bndtools/editor/pages/ExportAction.java

```diff
@@ -73,13 +74,13 @@ public void run(IProgressMonitor monitor) throws InvocationTargetException, Inte
 
         IFile targetResource = ResourceUtil.getFile(editor.getEditorInput());
         try {
-            Run run = LaunchUtils.createRun(targetResource);
+            Run run = LaunchUtils.createRun(targetResource, Mode.EXPORT);
 
             RunExportSelectionWizard wizard = new RunExportSelectionWizard(configElems, model, run);
             WizardDialog dialog = new WizardDialog(parentShell, wizard);
             dialog.open();
 
-            LaunchUtils.endRun(run);
+            LaunchUtils.endRun(run, Mode.EXPORT);
```

Comment:
thing is the `run` isn't what's passed into the RunListener, otherwise that would work. But it's a good point about the mode in both methods may result in errors if you get them wrong.